core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -209,13 +212,13 @@ class Main {
 
             val key = when (config.algorithm) {
                 EMConfig.Algorithm.MIO -> Key.get(
-                        object : TypeLiteral<MioAlgorithm<RestIndividual>>() {})
+                        object : TypeLiteral<MioAlgorithm<GraphqlIndividual>>() {})
                 EMConfig.Algorithm.RANDOM -> Key.get(
-                        object : TypeLiteral<RandomAlgorithm<RestIndividual>>() {})
+                        object : TypeLiteral<RandomAlgorithm<GraphqlIndividual>>() {})
                 EMConfig.Algorithm.WTS -> Key.get(
-                        object : TypeLiteral<WtsAlgorithm<RestIndividual>>() {})
+                        object : TypeLiteral<WtsAlgorithm<GraphqlIndividual>>() {})
                 EMConfig.Algorithm.MOSA -> Key.get(
-                        object : TypeLiteral<MosaAlgorithm<RestIndividual>>() {})
+                        object : TypeLiteral<MosaAlgorithm<GraphqlIndividual>>() {})
                 else -> throw IllegalStateException("Unrecognized algorithm ${config.algorithm}")
```

Comment:
doing something like this would make REST not usable anymore. this should depend on the `problemType`